figure.html

```diff
@@ -398,6 +398,131 @@
 </div> <!-- end #line-block -->
 
 <li>
+
+<p class="first">On the <span class="bold">Labels</span> tab click <span class="bold">Draw ROIs</span> to add ROI shapes to selected panes.</p>
+
+<p>In the ROIs window, draw rectangle, line, arrow and ellipse ROIs on the panel.</p>
+
+<p>Line colour and thickness can be selected using the drop-downs.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figureROI.jpg" class="align-center" src="images/figureROI.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li>
+
+<p class="first">ROI shapes can be edited or moved after adding, and copied and pasted.</p>
+
+<p>Click <span class="bold">OK</span> to save the shapes.</p>
+
+<p>Shapes can be copied in the Draw ROIs window and pasted directly onto panels in the main Figure drawing.</p>
```

Comment:
I would suggest exactly how to Paste : ``...in the main Figure drawing using shortcult Ctrl V or the Paste button in Labels tab. `` (note that the Paste in the Preview tab is not an option here, as it says "only rect. ROIs...."